integration-tests/cdi-app/cdi-app-standalone/pom.xml

```
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>jsf-app</artifactId>
```

It should probably be
```suggestion
        <artifactId>cdi-app</artifactId>
```

---

yeah right. For some reason it built on my machine but its defninitly wrong